docs/developer-guide.md

```diff
@@ -48,3 +48,67 @@ const showFlaggedFeature = useShowGrowthbookFeature('flagged-feature'); // Set t
 ### Cleanup
 
 Once a feature flag is no longer necessary, any code that was added (ex: `useShowGrowthbookFeature`) should be removed and the flag in Growthbook can be deleted.
+
+## Database Testing
+
+### pgTap + sqitch tips
+
+#### Run a single test
+
+You can run a single test by calling `pg_prove` directly from the command line like so:
+`pg_prove -v -d \<test_db_name\> \<path-to-test-file\>/file_to_test.sql`
```

Comment:
Line return for readability 

```suggestion
    ```
    pg_prove -v -d \<test_db_name\> \<path-to-test-file\>/file_to_test.sql`
    ```
```